src/Microsoft.Win32.Registry/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+// Registry tests can conflict with each other due to accessing the same keys/values in the registry
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly)]
```

Comment:
Isn't this the wrong hammer to use to fix these tests? Should we actually fix the tests and make them work in parallel? I'm fine doing this for now but we should have a issue tracking the removal if we don't already.